src/form/form.tsx

```diff
@@ -143,6 +144,11 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
                     <div className="col-xs-12">
                         {this.props.onCancel && this.getCancelButton()}
 
+                        {this.props.createButtonFactories &&
+                            this.props.createButtonFactories.map((createButtonFactory, index) =>
+                                createButtonFactory({ key: index, disabled: !this.state.isDirty })
```

Comment:
This is a implementation for your particular use case. If your use case was to disable your button on `state.isSubmitting` you have no chance to implement that. Please add this information to the buttonProps. It will also give us more flexibility in the future when additional context might be needed to render a button in a specific way.